src/librustc_privacy/lib.rs

```diff
@@ -347,19 +363,37 @@ impl<'a, 'tcx, 'v> Visitor<'v> for EmbargoVisitor<'a, 'tcx> {
                 }
             }
 
+            hir::ItemForeignMod(ref foreign_mod) => {
+                for foreign_item in &foreign_mod.items {
+                    let public = self.prev_public && foreign_item.vis == hir::Public;
+                    let exported = self.prev_exported && foreign_item.vis == hir::Public ||
+                                   self.reexports.contains(&foreign_item.id);
+
+                    if public { self.public_items.insert(foreign_item.id); }
+                    if exported { self.exported_items.insert(foreign_item.id); }
```

Comment:
I think stylistically in rustfmt it never formats `if` statements on one line, so to head off the churn could this go ahead and move to lines after braces?